scripts/app-stream/start-image-builder.js

```diff
@@ -0,0 +1,97 @@
+        this.imageBuilderName = `SWBImageBuilder-${Date.now()}`;
+    }
+
+    async run() {
+        try {
+            await this.createImageBuilder();
+            await this.waitForImageBuilderToBeReady();
+
+            const imageBuilderUrl = `https://console.aws.amazon.com/appstream2/home?region=${this.region}#/images?bottomTab=details&topTab=image-builders`
+
+            console.log(`You can find your new Image Builder at this address ${imageBuilderUrl}`)
+        } catch (e) {
+            process.exit(1);
+        }
+    }
+
+    async createImageBuilder() {
+        console.log(`Starting image builder with name ${this.imageBuilderName}`)
+        try {
+            const vpcsResponse = await this.ec2Client.send(
+                new DescribeVpcsCommand({
+                    Filters: [{
+                        Name: 'isDefault',
+                        Values: [true]
+                    }]
+                })
+            );
+            const vpcId = vpcsResponse.Vpcs[0].VpcId;
+            const subnetsResponse = await this.ec2Client.send(
+                new DescribeSubnetsCommand({
+                    Filters: [
+                        {
+                            Name: 'vpc-id',
+                            Values: [vpcId]
+                        }
+                    ]
+                })
+            )
+
+            const subnetId = subnetsResponse.Subnets[0].SubnetId;
+
+            await this.appStreamClient.send(
+                new CreateImageBuilderCommand({
+                        InstanceType: 'stream.graphics.g4dn.xlarge',
+                        Name: this.imageBuilderName,
+                        ImageName: 'AppStream-Graphics-G4dn-WinServer2019-06-01-2021',
+                        DisplayName: this.imageBuilderName,
+                        VpcConfig: {
+                            SubnetIds: [
+                                subnetId
+                            ]
+                        },
+                        EnableDefaultInternetAccess: true
+                    }
+                )
+            );
+        } catch (e) {
+            console.error("Failed to start image builder", e);
+            throw e;
+        }
+    }
+
+    async waitForImageBuilderToBeReady() {
+        try {
+            console.log("Waiting for Image Builder to be in RUNNNING state. This can take 5 to 10 minutes");
+            let imageInRunningState = false;
+            while (!imageInRunningState) {
+                await new Promise(r => setTimeout(r, 5000));
+                const decribeImageBuilderResponse = await this.appStreamClient.send(new DescribeImageBuildersCommand({
+                    Names: [
+                        this.imageBuilderName
+                    ]
+                }));
+                const imageState = decribeImageBuilderResponse.ImageBuilders[0].State;
+                console.log(`Image Builder is in ${imageState} state`)
```

Comment:
Probably display image builder name as well. Customer would need to enter this when setup AppStream via SWB UI.

Reply:
In the beginning of the script, it displays the name of the image. Would this be ok? Or should the script print the image name again at the end of the script. For reference this is the spot where the script prints the image name at the beginning of the script.
```
Starting Image Builder using AWS Profile thingut+swb-member-account in region us-east-1
Starting image builder with name SWBImageBuilder-1623942392638
Waiting for Image Builder to be in RUNNNING state. This can take 5 to 10 minutes
```

Reply:
Ah nvm, I just saw your other comment. It makes sense to log the image name at the end of the script also. I'll do that.